cli/sync.go

```diff
@@ -222,6 +234,10 @@ var SyncCheckpointCmd = &cli.Command{
 		defer closer()
 		ctx := ReqContext(cctx)
 
+		if cctx.NArg() != 1 {
```

Comment:
No, let's leave this one alone, it's a bit more subtle. Read the code below. If `--epoch` is set it tries to use that, otherwise it tries to parse the tipset from the rest of the args, which could be 1 or more - a tipset is made up of at least one CID and this command lets you pass in as many as required.